ops-bedrock/docker-compose.yml

```diff
@@ -42,11 +52,15 @@ services:
 
   op-node:
     depends_on:
+      - op_stack_go_builder
       - l1
       - l2
     build:
       context: ../
       dockerfile: ./op-node/Dockerfile
+      args:
```

Comment:
Are you sure this will work for local dev? ie you make the change locally and then want to run it in the devnet?